2015-07-20 22:43:37,593 - [INFO] - from play in pool-6-thread-5 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2015-07-20 22:43:45,750 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:mem:play

2015-07-20 22:43:46,281 - [ERROR] - from application in New I/O worker #1 


! @6mp3h63d5 - Internal server error, for (POST) [/logIn] ->

play.api.db.evolutions.InvalidDatabaseRevision: Database 'default' needs evolution![An SQL script need to be run on your database.]
	at play.api.db.evolutions.EvolutionsPlugin$$anonfun$onStart$1$$anonfun$apply$1.apply$mcV$sp(Evolutions.scala:510) ~[play-jdbc_2.11-2.3.9.jar:2.3.9]
	at play.api.db.evolutions.EvolutionsPlugin.withLock(Evolutions.scala:531) ~[play-jdbc_2.11-2.3.9.jar:2.3.9]
	at play.api.db.evolutions.EvolutionsPlugin$$anonfun$onStart$1.apply(Evolutions.scala:485) ~[play-jdbc_2.11-2.3.9.jar:2.3.9]
	at play.api.db.evolutions.EvolutionsPlugin$$anonfun$onStart$1.apply(Evolutions.scala:483) ~[play-jdbc_2.11-2.3.9.jar:2.3.9]
	at scala.collection.immutable.List.foreach(List.scala:381) ~[scala-library-2.11.6.jar:na]
	at play.api.db.evolutions.EvolutionsPlugin.onStart(Evolutions.scala:483) ~[play-jdbc_2.11-2.3.9.jar:2.3.9]
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91) ~[play_2.11-2.3.9.jar:2.3.9]
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:91) ~[play_2.11-2.3.9.jar:2.3.9]
	at scala.collection.immutable.List.foreach(List.scala:381) ~[scala-library-2.11.6.jar:na]
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:91) ~[play_2.11-2.3.9.jar:2.3.9]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91) ~[play_2.11-2.3.9.jar:2.3.9]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:91) ~[play_2.11-2.3.9.jar:2.3.9]
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21) ~[play_2.11-2.3.9.jar:2.3.9]
	at play.api.Play$.start(Play.scala:90) ~[play_2.11-2.3.9.jar:2.3.9]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:157) ~[play_2.11-2.3.9.jar:2.3.9]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130) ~[play_2.11-2.3.9.jar:2.3.9]
	at scala.Option.map(Option.scala:146) ~[scala-library-2.11.6.jar:na]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130) ~[play_2.11-2.3.9.jar:2.3.9]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128) ~[play_2.11-2.3.9.jar:2.3.9]
	at scala.util.Success.flatMap(Try.scala:230) ~[scala-library-2.11.6.jar:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128) ~[play_2.11-2.3.9.jar:2.3.9]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.9.jar:2.3.9]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.6.jar:na]

2015-07-20 22:43:47,940 - [WARN] - from play in play-internal-execution-context-1 
Error stopping plugin
java.lang.NullPointerException: null
	at play.libs.ws.ning.NingWSPlugin.onStop(NingWSPlugin.java:31) ~[play-java-ws_2.11-2.3.9.jar:2.3.9]
	at play.api.Play$$anonfun$stop$1$$anonfun$apply$1$$anonfun$apply$mcV$sp$2.apply(Play.scala:108) [play_2.11-2.3.9.jar:2.3.9]
	at play.api.Play$$anonfun$stop$1$$anonfun$apply$1$$anonfun$apply$mcV$sp$2.apply(Play.scala:107) [play_2.11-2.3.9.jar:2.3.9]
	at scala.collection.immutable.List.foreach(List.scala:381) [scala-library-2.11.6.jar:na]
	at play.api.Play$$anonfun$stop$1$$anonfun$apply$1.apply$mcV$sp(Play.scala:107) [play_2.11-2.3.9.jar:2.3.9]
	at play.api.Play$$anonfun$stop$1$$anonfun$apply$1.apply(Play.scala:107) [play_2.11-2.3.9.jar:2.3.9]
	at play.api.Play$$anonfun$stop$1$$anonfun$apply$1.apply(Play.scala:107) [play_2.11-2.3.9.jar:2.3.9]
	at play.utils.Threads$.withContextClassLoader(Threads.scala:21) [play_2.11-2.3.9.jar:2.3.9]
	at play.api.Play$$anonfun$stop$1.apply(Play.scala:106) [play_2.11-2.3.9.jar:2.3.9]
	at play.api.Play$$anonfun$stop$1.apply(Play.scala:105) [play_2.11-2.3.9.jar:2.3.9]
	at scala.Option.map(Option.scala:146) [scala-library-2.11.6.jar:na]
	at play.api.Play$.stop(Play.scala:105) [play_2.11-2.3.9.jar:2.3.9]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:142) [play_2.11-2.3.9.jar:2.3.9]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:130) [play_2.11-2.3.9.jar:2.3.9]
	at scala.Option.map(Option.scala:146) [scala-library-2.11.6.jar:na]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:130) [play_2.11-2.3.9.jar:2.3.9]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:128) [play_2.11-2.3.9.jar:2.3.9]
	at scala.util.Success.flatMap(Try.scala:230) [scala-library-2.11.6.jar:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:128) [play_2.11-2.3.9.jar:2.3.9]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) [play_2.11-2.3.9.jar:2.3.9]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) [scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) [scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) [scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library-2.11.6.jar:na]

2015-07-20 22:43:47,981 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:mem:play

2015-07-20 22:43:48,138 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-07-20 22:49:14,094 - [INFO] - from play in play-internal-execution-context-5 
database [default] connected at jdbc:h2:mem:play

2015-07-20 22:49:14,343 - [INFO] - from play in play-internal-execution-context-5 
Application started (Dev)

2015-07-20 22:49:29,245 - [INFO] - from play in play-internal-execution-context-8 
database [default] connected at jdbc:h2:mem:play

2015-07-20 22:49:29,313 - [INFO] - from play in play-internal-execution-context-8 
Application started (Dev)

2015-07-20 22:49:34,683 - [INFO] - from play in play-internal-execution-context-7 
database [default] connected at jdbc:h2:mem:play

2015-07-20 22:49:34,739 - [INFO] - from play in play-internal-execution-context-7 
Application started (Dev)

2015-07-20 22:51:25,623 - [INFO] - from play in play-internal-execution-context-9 
database [default] connected at jdbc:h2:mem:play

2015-07-20 22:51:25,680 - [INFO] - from play in play-internal-execution-context-9 
Application started (Dev)

2015-07-20 22:58:38,917 - [INFO] - from play in play-internal-execution-context-9 
database [default] connected at jdbc:h2:mem:play

2015-07-20 22:58:39,167 - [INFO] - from play in play-internal-execution-context-9 
Application started (Dev)

2015-07-20 23:07:28,845 - [ERROR] - from application in New I/O worker #15 


! @6mp3k91n9 - Internal server error, for (POST) [/logIn] ->

play.PlayExceptions$CompilationException: Compilation error[value nestUser is not a member of object controllers.Application]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:62) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:62) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.9.jar:2.3.9]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.9.jar:2.3.9]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.6.jar:na]

2015-07-20 23:08:13,034 - [ERROR] - from application in New I/O worker #15 


! @6mp3kbf6p - Internal server error, for (POST) [/logIn] ->

play.PlayExceptions$CompilationException: Compilation error[value nestUser is not a member of object controllers.Application]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:62) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:62) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.9.jar:2.3.9]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.9.jar:2.3.9]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.6.jar:na]

2015-07-20 23:08:30,387 - [ERROR] - from application in New I/O worker #15 


! @6mp3kceoa - Internal server error, for (POST) [/logIn] ->

play.PlayExceptions$CompilationException: Compilation error[value nestUser is not a member of object controllers.Application]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:62) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:62) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.9.jar:2.3.9]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.9.jar:2.3.9]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.6.jar:na]

2015-07-20 23:08:48,175 - [ERROR] - from application in New I/O worker #15 


! @6mp3kdf6e - Internal server error, for (POST) [/logIn] ->

play.PlayExceptions$CompilationException: Compilation error[too many arguments for method nextUser: ()play.api.mvc.Call]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:62) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:62) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.9.jar:2.3.9]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.9.jar:2.3.9]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.6.jar:na]

2015-07-20 23:09:03,159 - [ERROR] - from application in New I/O worker #15 


! @6mp3kebam - Internal server error, for (POST) [/logIn] ->

play.PlayExceptions$CompilationException: Compilation error[not enough arguments for method nextUser: (client: models.Client)play.mvc.WebSocket[String].
Unspecified value parameter client.]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:62) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:62) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.9.jar:2.3.9]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.9.jar:2.3.9]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.6.jar:na]

2015-07-20 23:09:17,447 - [ERROR] - from application in New I/O worker #15 


! @6mp3kf6ea - Internal server error, for (POST) [/logIn] ->

play.PlayExceptions$CompilationException: Compilation error[not found: type Client]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:62) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:62) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.9.jar:2.3.9]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.9.jar:2.3.9]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.6.jar:na]

2015-07-20 23:09:32,832 - [INFO] - from play in play-internal-execution-context-16 
database [default] connected at jdbc:h2:mem:play

2015-07-20 23:09:33,070 - [INFO] - from play in play-internal-execution-context-16 
Application started (Dev)

